### PR TITLE
Removes headsets and ASRS tablets from spawning in Valhalla

### PR DESCRIPTION
**Pull request by Wisemonster**
## About The Pull Request
Title
## Why It's Good For The Game
Valhalla marines shouldn't be getting access to headsets.
## Changelog
:cl:
fix: Fixed an issue with headsets and ASRS tablets in Valhalla
/:cl:

### CODE REVIEW

**Comment by Wisemonster** [comment]
Ok, turns out there was also a way to get an ASRS tablet, so I added a fix for that as well

**Review by ivanmixo — CHANGES_REQUESTED** [review]
Uh, what? This doesn't solve a thing. You're adjusting the outfits for the human dummies xenos can spawn, not the actual valhalla marines spawning in

**Comment by Wisemonster** [comment]
It's not a copypasta of the xeno button, because it's literally the xeno button but with a proc to allow added onto it, similar to how the vehicle spawner button works. I didn't make a new button, but rather just added onto the xeno button so marines could use in Valhalla and then renamed it (See #16212) The outfits allow for limited HvH testing until #16236 is merged.

File: code/game/objects/machinery/buttons.dm [hunk 1]
```diff
@@ -298,73 +299,55 @@
...
+		if(item.type in item_blacklist)
+			qdel(item) // Prevents blacklisted items from being spawned, like ASRS tablets and headsets
 
 /obj/machinery/button/valhalla/vehicle_button
 	name = "Vehicle Spawner"
 
-/obj/machinery/button/valhalla/vehicle_button/attack_alien(mob/living/carbon/xenomorph/xeno_attacker, damage_amount = xeno_attacker.xeno_caste.melee_damage, damage_type = BRUTE, armor_type = MELEE, effects = TRUE, armor_penetration = xeno_attacker.xeno_caste.melee_ap, isrightclick = FALSE)
-	var/list/spawnable_vehicles = list(/obj/vehicle/sealed/armored/multitile,
-	/obj/vehicle/sealed/armored/multitile/apc)
+/obj/machinery/button/valhalla/vehicle_button/proc/spawn_vehicles(mob/living/user)
```
Comment (ivanmixo) [inline code/game/objects/machinery/buttons.dm:331]: Forgor doc here as well

**Review by ivanmixo — APPROVED** [review]

File: code/game/objects/machinery/buttons.dm (hunk 1 above)
Comment (Wisemonster) [reply code/game/objects/machinery/buttons.dm:332]: Alright, gave it an autodoc

**Pull request merged**